overlord/snapstate/check_snap.go

```diff
@@ -53,6 +53,8 @@ var featureSet = map[string]bool{
 	"kernel-assets": true,
 	// Support for "refresh-mode: ignore-running" in snap.yaml
 	"app-refresh-mode": true,
+	// Support for "SNAP_UID" environment variable
+	"snap-uid-env": true,
```

Comment:
this probably should be snap-uid-envvar then, as we have a different entry that uses -env already to mean something a bit different

Reply:
Done.